gcloud/pubsub/api.py

```diff
@@ -52,3 +52,49 @@ def list_topics(page_size=None, page_token=None,
 
     path = '/projects/%s/topics' % project
     return connection.api_request(method='GET', path=path, query_params=params)
+
+
+def list_subscriptions(page_size=None, page_token=None, topic_name=None,
+                       project=None, connection=None):
+    """List subscriptions for a given project.
+
```

Comment:
Those links are fine but I literally meant using the resource names I included above. That's the way  to refer to these API methods. It's fine as is though.